schema/schema.sql

```diff
@@ -0,0 +1,75 @@
+CREATE DATABASE catalogue_of_things;
+
+CREATE TABLE books (
```

Comment:
The book should have a reference to the author as well as the genre. Kindly create the necessary foreign keys and columns with corresponding indexes.

Reply:
@muneebulrehman Thank you for your feedback. However, all your request are not part of the requirement and I would love to create them on my free time. thank you